### DISCUSSION TITLE
Add to Apollo subgraph compatibility test suite

### DISCUSSION THREAD

**Discussion by kdawgwilk** [99designs/gqlgen#1917] [Ideas]
Would be great to have compatibility determined by the new test suite from Apollo.  I have a PR up adding an example subgraph and would love any feedback to improve it

https://github.com/apollographql/apollo-federation-subgraph-compatibility/pull/42

**Comment by carldunham** [comment]
@kdawgwilk, how about just taking your implementations/gqlgen in that PR and dropping it into example/federation/products here? It won't pass until multiple @key directives are supported, of course.

**Comment by kdawgwilk** [comment]
I am okay with doing that but Apollo will likely want it in their test suite so we could have it in both places
